includes/Generator/Order.php

```diff
@@ -129,6 +124,26 @@ protected static function get_date_created( $assoc_args ) {
 		return $dates[ array_rand( $dates ) ];
 	}
 
+	/**
+	 * Returns a status to use as the order's status. If no status argument has been passed, this will
+	 * return a random status.
+	 *
+	 * @param array $assoc_args CLI arguments.
+	 * @return string An order status.
+	 */
+	private static function get_status( $assoc_args ) {
+		if ( ! empty( $assoc_args['status'] ) ) {
+			return $assoc_args['status'];
+		} else {
+			return self::random_weighted_element(array(
```

Comment:
Missing the whitespace before `array(`

Reply:
Thanks! I'm still getting used to the WP/WC conventions. 😅 

Fixed in 99e58edc7ad9cf50dce01a1ceb1868224667ebdc.